2025-09-10 23:37:15,304 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2025-09-10 23:37:15,309 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'delete']
2025-09-10 23:37:15,310 DEBUG    root            Running [gcloud.projects.delete] with arguments: [--quiet: "True", PROJECT_ID_OR_NUMBER: "lionsofzion-dev"]
2025-09-10 23:37:15,322 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2025-09-10 23:37:15,855 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "DELETE /v1/projects/lionsofzion-dev?alt=json HTTP/11" 403 None
2025-09-10 23:37:15,857 DEBUG    root            (gcloud.projects.delete) [admin@lionsofzion-official.org] does not have permission to access projects instance [lionsofzion-dev] (or it may not exist): The caller does not have permission. This command is authenticated as admin@lionsofzion-official.org which is the active account specified by the [core/account] property
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 946, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/surface/projects/delete.py", line 84, in Run
    result = projects_api.Delete(project_ref)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/api_lib/cloudresourcemanager/projects_api.py", line 170, in Delete
    client.projects.Delete(
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 732, in Delete
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 747, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 753, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 612, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpForbiddenError: HttpError accessing <https://cloudresourcemanager.googleapis.com/v1/projects/lionsofzion-dev?alt=json>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Wed, 10 Sep 2025 20:37:15 GMT', 'server': 'ESF', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'server-timing': 'gfet4t7; dur=284', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'transfer-encoding': 'chunked', 'status': 403}>, content <{
  "error": {
    "code": 403,
    "message": "The caller does not have permission",
    "status": "PERMISSION_DENIED"
  }
}
>
2025-09-10 23:37:15,859 ERROR    root            (gcloud.projects.delete) [admin@lionsofzion-official.org] does not have permission to access projects instance [lionsofzion-dev] (or it may not exist): The caller does not have permission. This command is authenticated as admin@lionsofzion-official.org which is the active account specified by the [core/account] property
